components/secondary_button/README.md

```diff
@@ -0,0 +1,90 @@
+---
+description: Secondary buttons allow users to carry out a supporting action
+title: Secondary Button
+status: Draft
+author: orinevares
+---
+
+![Status](https://img.shields.io/badge/Recommended-Draft-orange.svg)
+> Last Updated: March 27, 2019
+
+# Secondary Button
+Secondary buttons allow users to carry out a supporting action on your service, such as Back or Cancel.
+
+## Visual Example
+
+<component-preview path="components/secondary_button/sample.html" height="100px" width="800px"> </component-preview>
+
+## Use This For
+* A supporting action you want users to take on your site.
```

Comment:
These could be plural as there could be a few secondary actions, so maybe say "Supporting actions user can take on your site"